questionnaire/questionnaire.js

```diff
@@ -104,19 +104,19 @@ module.exports = {
                 },
                 'q-applicant-option': {
                     title: 'Select the option that applies to you',
                     type: 'string',
                     oneOf: [
                         {
                             title: 'Option 1: Sexual assault or abuse',
-                            const: 'option-1-sexual-assault-or-abuse'
+                            const: 'option-1:-sexual-assault-or-abuse'
```

Comment:
should these have a colon in them?

Reply:
This was simply so that it reads "Option 1: Sexual assault or abuse" on the summary page. Seemed like a quick fix, and didn't think it would cause an issue since it is just part of a string.

Reply:
Given this is a one off situation that would be more trouble than it's worth to programmatically insert a colon, I think this is ok. I cannot think of this being a situation where we have "shot ourselves in the foot" and regret it later.